.env.example

```
@@ -12,6 +12,11 @@ NODE_ENV=development
 DATABASE_URL=postgresql://USER:PASSWORD@localhost:5433/DB_NAME
 TEST_DATABASE_URL=postgresql://USER:PASSWORD@localhost:5433/DB_NAME_test
 
+# -- Docker Postgres (used by docker-compose services) -----------------------
+POSTGRES_USER=f3local
+POSTGRES_PASSWORD=f3local
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)